frontend/appflowy_flutter/lib/plugins/document/application/cover_image_picker_bloc.dart

```diff
@@ -0,0 +1,201 @@
+import 'dart:io';
```

Comment:
I have only one import of this file on my side. My be a merge issue?

Reply:
Ah, I've noticed that this file still exists. It is redundant and should be removed.

Reply:
![Screenshot 2023-05-16 at 12 30 01 PM](https://github.com/AppFlowy-IO/AppFlowy/assets/47111784/34f788fd-674a-4337-8fa2-61692fd7c01e)

@LucasXu0 i haven't worked on this file in this task and the file you are mentioning is imported only once on my side. There is also no warning for redundant imports otherwise it give warning when duplicate imports are found. There is clearly something wrong with the merging and git side.

Reply:
This file has been moved to another directory on the main branch. Obviously, some merge conflicts occurred when merging the main branch into your branch. Simply deleting it and pushing should be fine.

Reply:
are you sure i should just delete this file and push the code? The check will fail and the code won't be able to compile or run

Reply:
I helped to delete it and pushed the code. 

<img width="624" alt="Screenshot 2023-05-16 at 21 37 44" src="https://github.com/AppFlowy-IO/AppFlowy/assets/11863087/2f305ba3-48f9-4fc9-bad6-90f0633e1601">